### PR TITLE
Use placeholders for cached portals

### PR DESCRIPTION
**Pull request by MNoelJones**
This change does two things - firstly, use the accessor functions to get at the portal information (cached or otherwise), secondly, use default (placeholder) text for elements that haven't yet been retrieved for cached portals.

Further mods I'd like to make are to refresh the portal data as it becomes available (particularly when you select a link for an "UNCACHED" portal name, it would be nice to have it re-populate the portal list with the retrieved data)

### CODE REVIEW

File: plugins/portals-list.user.js [hunk 1]
```diff
@@ -67,9 +69,13 @@ window.plugin.portalslist.fields = [
     title: "Level",
     value: function(portal) { return portal.options.data.level; },
     format: function(cell, portal, value) {
+      var level_text = "L?";
```
Comment (michaeldever) [inline plugins/portals-list.user.js]: Perhaps a ternary might be neater here?

```javascript
var level_text = "L" + (value ? value : "?")
```

Comment (MNoelJones) [inline plugins/portals-list.user.js]: How obvious :D 
Will change that when I get a chance.

**Comment by mxxcon** [comment]
It seems like this PR contains too many unrelated changes..

**Comment by MNoelJones** [comment]
Hmm. Something odd's going on here; probably a misunderstanding on my part... I'll try and sort it all out and re-create the PR.

**Comment by mxxcon** [comment]
You created this PR from your `master`. So while you were adding more commits to your repo, they were automatically imported here. What you should've done is created a separate feature branch and make a pull request from it.

**Comment by MNoelJones** [comment]
Yes, I can see that, now! I had expected the pull to be taken at a particular revision.

**Pull request closed without merging**